README.md

```diff
@@ -108,16 +108,31 @@ When the upload is completed, click the **Data sets** tab and view the details o
-
-#### Connect to TAP Analytics Toolkit server and run model generation script
-1. Copy the contents from ‘src/main/atkmodelgenerator/atk_model_generator.py’ into your notebook.
-1. After the imports section, set the URI to the TAP Analytics Toolkit server: ‘ta.server.uri = <link-to-atk-server>’
-1. Set the value of ‘ds’ to be the link to the data set previously uploaded to HDFS (‘targetUri’).
-1. Run the script. The link to the created model in HDFS will be printed in the output.
+1. In Jupyter, click the (white) **Upload** button, then navigate to the Space Shuttle notebook file, which can be found here: `space-shuttle-demo-master/sparktkmodelgenerator/sparktk_shuttle_model_generator.ipynb`
+1. Select the file and click the second (blue) **Upload** button.
+1. In Jupyter, click on the `sparktk_shuttle_model_generator.ipynb` file to open the file.
+
+#### Connecting to TAP server and generating the model
+In this step, you modify the generic notebook script, then run it to create the model.
+1. In the TAP Console, navigate to the **Data catalog > Data sets** tab and then click on the name of the data set to show detailed information. Copy the URI for the data set for use in the next step.
+1. In Jupyter, paste this URI over the default name for data set `ds=` in the third cell.  
+```
+    ds=”your_dataset_uri_here”
+```  
+1. In Jupyter, select the second cell, then on the Jupyter menu, select **Cell > Run Cell, Select Below** (same as the ![](https://github.com/trustedanalytics/platform-wiki/blob/master/wikiImages/Jup_Run_Sel_Next_Icon.png) icon) to run the script in the cell.
+1. When execution of the cell is finished, the asterisk in the `In [*]` text on the right side of the cell is replaced by the cell number. Work through Cells 3 through 6,  one at a time, using **Cell > Run, Highlight Next**. Make sure you wait for the current cell to complete before moving to the next one.
+1. When you start Cell 7, you will be prompted for:  
+  - The URL of your TAP server. Paste the domain name of the TAP server into the field and press **Enter**. (Example: `ontap07.demo-gotapaas.com`.)  
+  - Your TAP user name. Type your user name and press **Enter**.  
+  - Your TAP password. Type your password and press **Enter**.  
+1. In the next cell, you are prompted for the organization number, for example **[1-2]**. Enter the organization number and press **Enter**.
+>If there is only one organization, you will *not* be prompted.
+
+1. Work through the remaining cells, one at a time, using **Cell > Run, Highlight Next**. Make sure you wait for the current cell to complete before moving to the next one.
+1. When finished, the trained model will appear in the **Data catalog > Data sets** tab. Filename = `spaceshuttleSVMmodel.mar`.
+>The Model ARchive format (.mar files) is a new model format for use with spark-tk.
+
+1. Click on the model name to see detailed information. Copy the targetURI for the model for use in Step 5.
```

Comment:
The "Step 5" makes sense in the Wiki source text because the major steps are numbered. But here you should put in the title of the heading you are referring to.